store/src/main/java/org/apache/rocketmq/store/DefaultMessageStore.java

```diff
@@ -782,10 +783,20 @@ private String getStorePathPhysic() {
     public HashMap<String, String> getRuntimeInfo() {
         HashMap<String, String> result = this.storeStatsService.getRuntimeInfo();
 
-        {
-            double physicRatio = UtilAll.getDiskPartitionSpaceUsedPercent(getStorePathPhysic());
+        String commitLogStorePath = DefaultMessageStore.this.getMessageStoreConfig().getStorePathCommitLog();
+        if (commitLogStorePath.contains(MessageStoreConfig.MULTI_PATH_SPLITTER)) {
+            double maxValue = Double.MIN_VALUE;
```

Comment:
Is `maxPhysicsUsedRatio` more appropriate?

Reply:
@caigy Yes, `maxPhysicsUsedRatio` is better .
But after some consideration, I think we should use `minPhysicsUsedRatio`. It's more useful for admins, because we use min value of all store path to determine whether disk is full and we should clean disk immediately.

Reply:
@Jason918 I agree. Using `minPhysicsUsedRatio` is more suitable for this scenario.